NvAPIWrapper/Display/PathInfo.cs

```diff
@@ -21,7 +21,7 @@ public class PathInfo : IEquatable<PathInfo>
         /// <param name="resolution">Display resolution</param>
         /// <param name="colorFormat">Display color format</param>
         /// <param name="targetInfos">Target configuration informations</param>
-        public PathInfo(Resolution resolution, ColorFormat colorFormat, PathTargetInfo[] targetInfos)
+        public PathInfo(Resolution resolution, Format colorFormat, PathTargetInfo[] targetInfos)
```

Comment:
Changing `ColorFormat` to `Format` breaks backward compatibility and forces a new major version. Isn't this possible to change the new type to something like `HDRColorFormat`?

Reply:
Done!

Reply:
Thanks